style.css

```diff
@@ -470,3 +491,144 @@ section .contact_forms .e_mail {
   color: #42526e;
   padding-bottom: 10px;
 }
+
+@media (min-width: 800px) {
+  body {
+    background-image: url(assets/background.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-origin: content-box;
+    background-size: cover;
+  }
```

Comment:
- [ ] [_Optional_] I think you should also explore `background-size: 100% 100%` and see how it turns out. The problem with cover is that the background starts disappearing when it's too low or too high a screen width, but this is entirely upto you and merely a suggestion.